blocks/table-v2/LICENSE-APACHE.md

```diff
@@ -0,0 +1,191 @@
+<!-- markdownlint-disable -->
+
```

Comment:
As part of this PR, please remove the old `blocks/table` folder and replace it with this (renaming this from `table-v2` to just `table` in the process).

We don't want the old table code (targeting an unsupported version of the spec) hanging around any more.

Reply:
https://github.com/hashintel/hash/pull/1946/commits/2048c0c2395c40a93e49db706cb444763e379794 done here